modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1911,7 +1910,15 @@ void setViewportLength(double value) {
         this.absoluteOffset = getPosition() * (estimatedSize - viewportLength);
         recalculateEstimatedSize();
     }
-    double getViewportLength() {
+
+    /**
+     * The length of the viewport portion of the VirtualFlow as computed during the layout pass.
+     * For a vertical flow, this corresponds to the height and for a horizontal flow to the width of the clip view,
+     * but it does not necessarily have to be the same value.
+     *
+     * @return the viewport length in pixel
```

Comment:
That should be "in pixels" (plural).

Also, please add an `@since 23` tag after the return tag.

Reply:
Done.